### DISCUSSION TITLE
Is it possible to highlight a returning user's first message of the stream?

### DISCUSSION THREAD

**Discussion by MC31415926535897** [Chatterino/chatterino2#4189] [Q&A] [answered]
I was wondering if it was possible to highlight a user's first message of a stream. Not their first message in the channel in all time, but their first message since the channel has most recently gone live. A little like how a Twitch Chat Pet welcomes users on stream when a user types in their first message since the last time the browser source was loaded, but just highlighting the chat message in chatterino. If so, how may I do this?

**Comment by Felanbird** [comment]
Not currently, I know that Twitch was experimenting with this feature, but I have no idea in what way they implemented it, and the person who would likely know/have an example, is on vacation. So I cant give you a definitive yes/no on whether or not it's possible for us to add it.

**Reply by iProdigy** [reply]
`returning-chatter=1` is the twitch experiment

**Reply by Felanbird** [reply]
Then while it is not currently possible, it could be in the future.

**Comment by pajlada** [comment, marked as answer]
Potentially possible to implement yourself as a plugin - see https://github.com/Chatterino/chatterino2/discussions/4999 for plugin idea discussions.
